### ISSUE TITLE
Can I export historical data with this lib

### ISSUE THREAD

**Issue by zdavatz** [Barabbas0/bitfinex-api-python#1]
Hi

are you intending to be able to export historical data with this library using authentication?

Best
Zeno

**Comment by Barabbas0** [comment]
Not yet. For that, though, I'd probably recommend a different API service
specializing in historical crypto data.

**Comment by zdavatz** [comment]
Can you recommend a library that works? I need the historical data of my accounts not of publicly available data.

**Comment by Barabbas0** [comment]
I could probably make this work. I basically have added functionality as I've needed it. What are you looking for, past trades?

**Comment by zdavatz** [comment]
Yes, correct. What one needs for Tax reasons is all currencies and all wallets on a monthly basis. See: https://github.com/zdavatz/rapflag/blob/master/rapflag.rb
